packages/gestalt/src/Typography.css

```diff
@@ -75,7 +75,11 @@
 /* overflow */
 
 .breakWord {
-  word-wrap: break-word;
```

Comment:
word-wrap: break-word; & word-break: break-word; are equivalent. can we add that to the description?

https://www.w3schools.com/cssref/playdemo.php?filename=playcss_word-break&preval=break-word
https://www.w3schools.com/cssref/playdemo.php?filename=playcss_word-wrap&preval=break-word

Reply:
I don't think we should use word wrap, it's actually a deprecated property:
https://stackoverflow.com/questions/17259916/difference-between-overflow-wrap-and-word-break